README.md

```diff
@@ -2,29 +2,104 @@
 
 ## Getting Started
 
-These sections will describe required or recommended steps so that your Ansible Automation Platform deployment is as seamless as possible.
+This section will walk through deploying the AWS infrastructure and Ansible Automation Platform.
 
+You may also download the this repository from GitHub and modify to suit your needs.
 
 ### Red Hat Enterprise Linux
 
 You will need to use a Red Hat Enterprise Linux (RHEL) Amazon Machine Image (AMI) as the foundation for your deployment.  While this collection will automatically find a public RHEL AMI available from AWS, public images bill for RHEL outside of your subscription for Ansible Automation Platform.
 
 It is recommended that you create a custom AMI that you may then use to deploy RHEL with your subscriptions that come with Ansible Automation Platform.  [Red Hat Image Builder][image-builder] is a utility that makes creating a custom AMI easy.
 
+### AWS Credentials
+
+This terraform template requires AWS credentials for deploying infrastructure, which can be set by running command,
+`aws configure`
+
+This template will need a way to connect to the virtual machines that it creates.
+By default, VMs are created with public IP addresses to make this simple, but the template may be modified to use private IP addresses if your local machine can route traffic to private networks.
 
 ## Deploying Ansible Automation Platform
 
 This section will walk through deploying the AWS infrastructure and Ansible Automation Platform.
 
+**NOTE:** This template is designed to provide a quick and easy setup with default values for the number of instances and instance types. However, these values can be customized to meet your specific requirements. 
+
+### Checklist
+
+- [ ] Install this repository
+- [ ] Terraform installed locally (`terraform`)
+- [ ] A RHEL AMI (if not using hourly RHEL instances)
+- [ ] A locally downloaded copy of the [AAP installer][aap-installer]
+- [ ] A variables file configured with required variables
+- [ ] Configure the AWS environment variables for authentication
+- [ ] An inventory file with the proper SSH configuration
+
 ### Deploying Infrastructure
 
+The variables below are required for running this template
+
+| Variable | Description |
+| -------- | ----------- |
+| `aap_red_hat_username` | This is your Red Hat account name that will be used for Subscription Management (https://access.redhat.com/management). |
+| `aap_red_hat_password` | The Red Hat account password. |
+| `infrastructure_db_username` | Username that will be the admin of the new database server. |
+| `infrastructure_db_password` | Password of the admin of the new database server. |
+| `aap_admin_password` | The admin password to create for Ansible Automation Platform application. |
+| `infrastructure_controller_ami` | The AMI for controller. In this template currently this value set to "" as default in which case it will pick RHEL AMI automatically. |
```

Comment:
Please move the *_ami variable info to the optional section since they now default to ""